pkgs/tools/security/fcrackzip/default.nix

```diff
@@ -0,0 +1,25 @@
+{stdenv, fetchurl}:
+
+stdenv.mkDerivation rec {
+  name = "fcrackzip-${version}";
+  version = "1.0";
+  src = fetchurl {
+    url = "http://oldhome.schmorp.de/marc/data/${name}.tar.gz";
+    sha256 = "0l1qsk949vnz18k4vjf3ppq8p497966x4c7f2yx18x8pk35whn2a";
+  };
+
+  patches = [ ./fcrackzip_forkexec.patch ];
```

Comment:
If possible could you add a comment above the `patches` line explaining what the patch does?